src/components/Toolbox/index.tsx

```diff
@@ -0,0 +1,184 @@
+                        <div className="hole"></div> - <span className="hole-toolbox"></span>
+                    </div>
+                    <div id="add-bin-mul-expr-btn" className="button">
+                        <div className="hole"></div> * <span className="hole-toolbox"></span>
+                    </div>
+                    <div id="add-bin-div-expr-btn" className="button">
+                        <div className="hole"></div> / <span className="hole-toolbox"></span>
+                    </div>
+                </div>
+    }
+
+    addBooleans(){
+        return  <div className="group">
+                    <p>Booleans</p>
+                    <div id="add-bin-and-expr-btn" className="button">
+                        <div className="hole"></div> and
+                        <div className="hole"></div>
+                    </div>
+                    <div id="add-bin-or-expr-btn" className="button">
+                        <div className="hole"></div> or <span className="hole-toolbox"></span>
+                    </div>
+                    <div id="add-unary-not-expr-btn" className="button">not <div className="hole"></div></div>
+                </div>
+    }
+
+    addComparisons(){
+        return <div className="group">
+                    <p>Comparisons</p>
+                    <div id="add-comp-eq-expr-btn" className="button">
+                        <div className="hole"></div> ==
+                        <div className="hole"></div>
+                    </div>
+                    <div id="add-comp-neq-expr-btn" className="button">
+                        <div className="hole"></div> !=
+                        <div className="hole"></div>
+                    </div>
+                    <div id="add-comp-lt-expr-btn" className="button">
+                        <div className="hole"></div> &lt;
+                        <div className="hole"></div>
+                    </div>
+                    <div id="add-comp-lte-expr-btn" className="button">
+                        <div className="hole"></div> &lt;=
+                        <div className="hole"></div>
+                    </div>
+                    <div id="add-comp-gt-expr-btn" className="button">
+                        <div className="hole"></div> &gt;
+                        <div className="hole"></div>
+                    </div>
+                    <div id="add-comp-gte-expr-btn" className="button">
+                        <div className="hole"></div> &gt;=
+                        <div className="hole"></div>
+                    </div>
+                </div>
+    }
+
+    addUserDefinedVariables(){
+        return  <div id="user-variables">
+                    <h2 id="user-defined-vars-heading">User-Defined Variables</h2>
+                    <div id="vars-button-grid">
+                    </div>
+                </div>
+    }
+
+    buildToolbox(){
+        return <div id = "mainToolboxDiv" className = {this.props.className}>
```

Comment:
Let me know if you want a different way of loading the toolbox. This is just a simple option check against a json (`./options.json`).